### ISSUE TITLE
Fireguard destroyers stuck in lava, not moving

### ISSUE THREAD

**Issue by monkfish** [the-hyjal-project/bugtracker#134]
**Describe Your Issue**
Inside the black rock depths instance, when you do the "lava run"  ( trick that consists on jumping down into lava on incendius and just moving from there to the molten bridge / black forge / MC entrance )
there are some fireguard destroyers in there in the middle of the lava that are bugged.
they will aggro you and use ranged fireballs. but they wont move from their spot, so after youre out of range for a while they will just reset, and if youre not exactly in front of them they will evade

The fireguard destroyers that are on top of little rocks will aggro properly and come until they reach the lava, then they will get stuck and pathing is all messy, sometimes they will reset and sometimes they will follow you clipping through textures
 
 TL;DR i think the fireguard destroyers pathing / behavior in lava is buged.
 
you can see here : 
minute 26:00+
minute 27:35+
https://www.twitch.tv/videos/843298026

Also another thing is theyre completely underneath the lava you cannot see them. in other servers i think they would be above the lava and easy to spot

**Steps To Reproduce**

1. Go inside black rock depths, to incendius lair, then jump down into lava, then do the lava run, you will see
2. Theres a few fireguard destroyers around, some inside the lava and others on top of little rocks 
3. you should try to pull them and see their behavior you will see they act weird

NPC: https://classic.wowhead.com/npc=8911/fireguard-destroyer

**Expected Behavior**

They should be patrolling above the lava and have correct pathing, if pulled they would come to you and attack after performing the ranged fireball spell.

**Comment by Nolin7777** [comment]
Hack: Not force to use MMaps?
